packages/blockchain-api/src/transaction/TransactionNavigator.ts

```diff
@@ -0,0 +1,101 @@
+import { BlockscoutCeloTransfer } from '../blockscout'
+import { ContractAddresses, Contracts } from '../utils'
+import { TransferCollection } from './TransferCollection'
+
+const MINTED_TOKEN_ADDRESS = '0x0000000000000000000000000000000000000000'
+
+export class TransactionNavigator {
```

Comment:
Thinking this could be renamed, maybe `TransfersNavigator`?

Reply:
Makes sense to me, done.